### ISSUE TITLE
DatePicker Background Color

### ISSUE THREAD

**Issue by sakinmuhendis** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#1182]
Hi everyone, I just need to add a datePicker to my project. However, when I change the background of the MainWindow, datePickers month and year pages does not show. Following images explain the issue. How can I solve it? 
![2019-01-31_20h12_59](https://user-images.githubusercontent.com/28702155/52072071-581a8a80-2595-11e9-80b9-0ccd77b57734.png)
![2019-01-31_20h13_17](https://user-images.githubusercontent.com/28702155/52072076-594bb780-2595-11e9-9e91-46633f5f6891.png)

**Comment by jespersh** [comment]
It is set to `PrimaryHueMidBrush`

https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.Calendar.xaml#L400

**Comment by sakinmuhendis** [comment]
I realized that I asked my question not that clear. 
Firstly, I was trying to add basic DatePicker that everyone can find from the Toolbox. However, I encountered a problem which was explained in the starting question of mine.
Secondly, I found that there is a ResourceDictionary in material design for Calendar design, and I said to myself I should be able to use it. But this time, I realized I do not know how to write the xmal definition for it. 
At this point I only added the following snippet
 `<ResourceDictionary Source="pack://application:,,,/MaterialDesignThemes.Wpf;component/Themes/MaterialDesignTheme.Calendar.xaml" />`
 to the App.xaml file. 
And now, I hope to add a definition like I did for the TimePicker which is: 
`<materialDesign:TimePicker HorizontalAlignment="Left" VerticalAlignment="Top" Is24Hours="True" Margin="182,443.5,0,0"  Style="{StaticResource MaterialDesignFloatingHintTimePicker}"  FontSize="17" />
`
How can I do that for calendar definition.
I hope I did explain myself more clear. Thanks.. @jespersh @Keboo
